__tests__/LottoResultTest.js

```diff
@@ -0,0 +1,17 @@
+import Lotto from '../src/Model/Lotto';
+
+const goalLotto = new Lotto([1, 2, 3, 4, 5, 6]);
+const bonusNumber = 7;
+
+describe('당첨 결과 테스트', () => {
+  test('맞는 번호 개수 및 보너스 번호가 포함됐는지 확인', () => {
+    const lottoArray = [new Lotto([1, 2, 3, 4, 5, 6]), new Lotto([1, 2, 3, 4, 7, 8]), new Lotto([1, 2, 3, 4, 5, 8])];
```

Comment:
이건 사소한 거긴 하지만, 변수명으로 자료형을 적게 되면 이후에 자료형이 변할 경우 변수명도 수정해야 하는 번거로움이 있을 수 있다고 하더라구요 🙂 예를 들면, array를 set형태로 바꾸면 lottoSet으로 바꾸어야 하는 것처럼 말이죠. 배열을 표현하는 변수명으로 lottos와 같은 복수형 변수명은 어떠신가요?
```suggestion
    const lottos = [new Lotto([1, 2, 3, 4, 5, 6]), new Lotto([1, 2, 3, 4, 7, 8]), new Lotto([1, 2, 3, 4, 5, 8])];
```